services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/RateDetailsSummarySectionV2.tsx

```diff
@@ -269,14 +275,29 @@ export const RateDetailsSummarySectionV2 = ({
                             </h3>
                             <dl>
                                 <DoubleColumnGrid>
+                                    {rate.formData.deprecatedRateProgramIDs
```

Comment:
The rate summary page doesn't show historical programs. Is that something needed on the page?

Reply:
@JasonLin0991 were you able to test the historic submission part in the review app? Or was this local? I'm working on a way for Meghan to be able to test this pr

Reply:
I did not test that page. I assumed since it was using the same `RateDetailsSummarySectionV2` it would be the same. Let me know if you need me to take a peek at it.